CONTRIBUTING.md

```diff
@@ -51,6 +51,26 @@ yarn workspaces foreach -Rt --no-private run build
 
 - `yarn reset` - will clean all `node_modules`, re-install dependencies and link them, and then build all JSS packages.
 
+## Developing sample
+
+For the creation of the samples we are using `initializers` approach. 
+
+- `initializer` - the process for handling the creation of the appropriate template. See [initializers](https://github.com/Sitecore/jss/tree/dev/packages/create-sitecore-jss/src/initializers).
+- `template` - the sample populated by [ejs](https://ejs.co/) tokens. See [templates](https://github.com/Sitecore/jss/tree/dev/packages/create-sitecore-jss/src/templates). Templates can be:
+	- `base` - the template that contains foundation for the application (e.g. *nextjs*).
+	- `feature` - the template that provides specific feature for the base template. Multiple *feature* templates can be applied to the *base* (e.g. *nextjs-styleguide*).
+
+If you want to use [*create-sitecore-jss*](https://github.com/Sitecore/jss/tree/dev/packages/create-sitecore-jss) from your local repository, run:
+
+```
+cd ./packages/create-sitecore-jss
+npm i -g
+```
+
+To start developing the sample you have to do next:
+* enable [watch.json](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/watch.json.example) file. You have to set up appropriate arguments for the sample which you want to start to develop
```

Comment:
We should maybe specifically instruct them to _make a copy_ of the example and name it "watch.json" (instead of implied rename).